projects/spectator/package.json

```diff
@@ -29,9 +29,10 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": ">= 10.1.0",
-    "@angular/core": ">= 10.1.0",
-    "@angular/router": ">= 10.1.0",
+    "@angular/common": ">= 11.1.0",
+    "@angular/core": ">= 11.1.0",
+    "@angular/router": ">= 11.1.0",
+    "@angular/cdk": ">= 11.1.0",
```

Comment:
We don't need to cdk here.

Reply:
Yeah this is not needed. Do you want to keep the other peer dep bump?